api/src/test/java/org/openmrs/notification/MessageServiceTest.java

```diff
@@ -113,4 +118,107 @@ public void sendMessage_shouldSendMessage() throws MessageException {
 		}
 	}
 	
+	/**
+	 * @see MessageService#getMessagePreparator()
+	 */
+	@Test
+	public void getMessagePreparator_shouldGetMessagePreparator() {
```

Comment:
remove this one, there is no logic in the getter. we dont test such getters. same applies for setters.